sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -301,16 +307,16 @@ abstract class SessionCatalogSuite extends AnalysisTest with Eventually {
       val tempTable2 = Range(1, 20, 2, 10)
       catalog.createTempView("tbl1", tempTable1, overrideIfExists = false)
       catalog.createTempView("tbl2", tempTable2, overrideIfExists = false)
-      assert(catalog.getTempView("tbl1") == Option(tempTable1))
-      assert(catalog.getTempView("tbl2") == Option(tempTable2))
-      assert(catalog.getTempView("tbl3").isEmpty)
+      assert(getTempViewRawPlan(catalog.getTempView("tbl1")) == Option(tempTable1))
```

Comment:
Shall we test the actual plan with the View wrapper? To make sure the generated `CatalogTable` is correct.